lib/multiplexer/multiplexer.go

```diff
@@ -94,17 +95,33 @@ func New(cfg Config) (*Mux, error) {
 		return nil, trace.Wrap(err)
 	}
 
+	entry := log.WithFields(log.Fields{
+		trace.Component: teleport.Component("mx", cfg.ID),
+	})
+	logLimiter, err := loglimit.New(loglimit.Config{
+		Entry:         entry,
+		LogSubstrings: errorSubstrings,
+		// ChannelSize is set to 0 (creating an unbuffered channel),
+		// since the channel is only written to when the goroutine
+		// responsible for a new connection is about to conclude its
+		// work (because an error was detected) and thus, if the
+		// sending blocks, no subsequent work is delayed.
+		ChannelSize: 0,
```

Comment:
Suggestion:

```suggestion
```

I'd also consider dropping the field from the Config object, since we don't really need it.

Reply:
Yes, let's only add what we need for now.

Reply:
Removed in 833458547eb2aac5fcabf4ce3c7cb28ae37aacc4.